### DISCUSSION TITLE
Sonoff NSPanel is ESP32 based, ping me hardware questions

### DISCUSSION THREAD

**Discussion by notenoughtech** [arendst/Tasmota#13306] [Ideas]
I really hope NSPanel will be supported, as the cat is out of the bag now, I'm happy to answer any hardware questions that could help developing tasmota on it:
more info: [https://notenoughtech.com/featured/sonoff-nspanel/](https://notenoughtech.com/featured/sonoff-nspanel/)

**Comment by sfromis** [comment]
The first question is probably details about what panel and type of touch control it is.

Tasmota already has support for the similar [Lanbon L8](https://templates.blakadder.com/lanbon_L8-HS.html).

(Just to get the ball rolling, I'm not myself going into the development which may be needed)

**Comment by dacCR** [comment]
Can it be used to display text and images from MQTT? I know it's not a display specifically but I wondered if it can be used in other ways such as this.
